pyproject.toml

```diff
@@ -90,6 +90,7 @@ dev = [
     "coverage[toml]>=7.10.3",
     "dirty-equals>=0.9.0",
     "duckduckgo-search>=7.0.0",
+    "fastmcp>=2.12.0",
```

Comment:
I don't think we need it here if it's already coming in through `pydantic-ai-slim[...,fastmcp,...]` above